test/jdk/java/awt/Window/Grab/GrabTest.java

```diff
@@ -100,6 +116,7 @@ public void mousePressed(MouseEvent e) {
 
         f.setVisible(true);
         w.setVisible(true);
```

Comment:
Since you are already touching this test, all the GUI work can be moved to the EDT.